### DISCUSSION TITLE
Tr

### DISCUSSION THREAD

**Discussion by meemijung** [community/community#135777] [Programming Help]
### Select Topic Area

Question

### Body

import tron from '@api/tron';

tron.getAssetByIdOrIssuer({only_confirmed: 'false', identifier: 'TYj71BXy8csT7sfVUzQ9gAj81Ts3NNTjEi'})
  .then(({ data }) => console.log(data))
  .catch(err => console.error(err));

**Comment by erencanbas** [comment]
getAssetByIdOrIssuerMake sure that the parameters accepted by the function are correct. For example, if 'only_confirmed'the parameter 'false'is written as , it is passed as a string ( 'false'). If this parameter expects a boolean (logical) value, it should be passed 'false'as false

`tron.getAssetByIdOrIssuer({ only_confirmed: false, identifier: 'TYj71BXy8csT7sfVUzQ9gAj81Ts3NNTjEi' })`

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @meemijung ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
